src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -118,13 +118,15 @@ def generate_initial_population(self, pop_data: SimulantData) -> None:
         pop["housing_type"] = pop["housing_type"].astype(
             pd.CategoricalDtype(categories=data_values.HOUSING_TYPES)
         )
+        # set int dtypes to float64 to play nicely with population_view checks
```

Comment:
Maybe I just lack context, but what do you mean by this? It seems a bit odd to store state and puma as floats.

Reply:
As discussed, it's because the population_view.update method breaks because they somehow get converted to floats while processing. I'll likely put this back to integers (which it really should be) and then coerce to floats where appropriate.

All the more reason to implement a more robust schema system